plasma_framework/contracts/src/transactions/PaymentTransactionModel.sol

```diff
@@ -13,7 +13,6 @@ library PaymentTransactionModel {
     uint8 constant public MAX_OUTPUT_NUM = 4;
 
     uint8 constant private ENCODED_LENGTH_WITH_METADATA = 4;
```

Comment:
Since there no longer is `ENCODED_LENGTH_WITHOUT_METADATA`, you can simple rename this constant into `ENCODED_LENGTH`